sigs.yaml

```diff
@@ -1571,10 +1571,22 @@ sigs:
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/build/debian-hyperkube-base/OWNERS
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/cmd/hyperkube/OWNERS
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/images/hyperkube/OWNERS
-    - name: release
+    - name: release-team
+      description: >
+        The Kubernetes Release Team is responsible for the day to day work
+        required to successfully create releases of Kubernetes.
       owners:
       - https://raw.githubusercontent.com/kubernetes/release/master/OWNERS
+      - https://raw.githubusercontent.com/kubernetes/sig-release/master/release-team/OWNERS
```

Comment:
sig-release seems to be its own subproject?

Reply:
deeper OWNERS files take precedence, the set of people in charge of the release team may be different from the set of people in charge of sig release

kuberentes/kubernetes is full of all sorts of prior art for this, but maybe that's seen as a one off

I have a similar view for eg: kubernetes/test-infra vs. kubernetes/test-infra/prow

Reply:
My bad, I just read this as "only sig-release". :P